### ISSUE TITLE
Push not work when app is stopped in Android 6

### ISSUE THREAD

**Issue by fattanasio** [phonegap/phonegap-plugin-push#1476]
Hi,
In Android 5, if you closed the app or you restarted the phone, notifications work properly but in Android 6 not work..
Perhaps it is not handled automatically restart services intent ?
Help me ,Thank you!

**Comment by macdonst** [comment]
Sorry to be a jerk but I'm now closing issues that don't follow the issue submission guidelines. This plugin takes a lot of work to maintain and I can't afford the time required to solicit the required info from everyone.

I'm closing this issue but please feel free to re-open the issue if you provide the requested details from the new issue submission guidelines.

https://github.com/phonegap/phonegap-plugin-push/blob/master/.github/ISSUE_TEMPLATE.md

**Comment by fattanasio** [comment]
Hi, thank you but I would understand if it is a known bug.


________________________________
Da: Simon MacDonald <notifications@github.com>
Inviato: martedì 20 dicembre 2016 16.02
A: phonegap/phonegap-plugin-push
Cc: fattanasio; Author
Oggetto: Re: [phonegap/phonegap-plugin-push] Push not work when app is stopped in Android 6 (#1476)


Sorry to be a jerk but I'm now closing issues that don't follow the issue submission guidelines. This plugin takes a lot of work to maintain and I can't afford the time required to solicit the required info from everyone.

I'm closing this issue but please feel free to re-open the issue if you provide the requested details from the new issue submission guidelines.

https://github.com/phonegap/phonegap-plugin-push/blob/master/.github/ISSUE_TEMPLATE.md

-

**Comment by macdonst** [comment]
@fattanasio you are not giving me enough information to debug this issue. Please fill out the issue guidelines.

**Comment by fattanasio** [comment]
Sorry but i didn't understand. My english is not very good :)

**Expected Behaviour**
When I stopped the App (kill from memory or restart smartphone) I would like to receive notification
**Actual Behaviour**
When the is opened i receive push notifications but when I stopped the App (kill from memory or restart smartphone) doesn't  receive push notifications
**Reproduce Scenario (including but not limited to)**
**Steps to Reproduce**
On Android 6 kill app and try to receive push notifications. It doesn't work. It works on Android 5
**Platform and Version (eg. Android 5.0 or iOS 9.2.1)**
Android 6.0, Tested on Asus Zen Fone Max and on Huawey P8
Cordova CLI version and cordova platform version
Cordova 6.4
Plugin version
1.9.0
Sample Push Data Payload
Sample Code that illustrates the problem
Logs taken while reproducing problem
nothing

Thank you
